portal/server/controllers/utilisation-report-service/record-correction/provide-utilisation-report-correction/helpers.ts

```diff
@@ -90,20 +93,47 @@ export const getAdditionalCommentsFieldLabels = (correctionReasons: RecordCorrec
 };
 
 /**
- * Maps the saved form values to the form values view model.
- * @param savedFormValues - The saved transient form values
+ * Maps a monetary value string to a monetary form value.
+ *
+ * If the value is undefined, returns null.
+ * If the value is a valid monetary value, it will be formatted with two
+ * decimal places and thousands separators.
+ * If the value is invalid, returns the original string.
+ * @param monetaryValue - The monetary value to format
+ * @returns Formatted monetary value string, null if undefined, or original string if invalid
+ */
+export const mapMonetaryValueToProvideCorrectionFormValue = (monetaryValue?: string | number): string | null => {
```

Comment:
I also think this name is wrong, there is a very realistic chance the input is not a monetary value here, e.g. the test case 'INVALID'